File: Beecrowd/2310 Voleibol/main.py
n = int(input())
res = []
res1 = 0
res2 = 0
res3 = 0
res4 = 0
res5 = 0
res6 = 0

for _ in range(n):
    str = input()
    s, b, a = [int(_) for _ in input().split()]
    s1, b1, a1 = [int(_) for _ in input().split()]

    res1 += s
    res2 += b
    res3 += a
    res4 += s1
    res5 += b1
    res6 += a1

res.insert(0, res4 / res1 * 100)
res.insert(1, res5 / res2 * 100)
res.insert(2, res6 / res3 * 100)

print("Pontos de Saque: %.2f %%."  %res[0])
print("Pontos de Bloqueio: %.2f %%." %res[1])
print("Pontos de Ataque: %.2f %%." %res[2])

File: Beecrowd/2310 Voleibol/main.cpp
#include <iostream>

using namespace std;

int main(){
    int n, s, b, a, s1, b1, a1, res1[6] = {0, 0, 0, 0, 0, 0};
    double res[3];
    string str;

    cin >> n;

    while(n--){

        cin >> str;
        cin >> s >> b >> a;
        cin >> s1 >> b1 >> a1;

        res1[0] += s;
        res1[1] += b;
        res1[2] += a;
        res1[3] += s1;
        res1[4] += b1;
        res1[5] += a1;

    }

    res[0] = (double)res1[3] / res1[0] * 100.0;
    res[1] = (double)res1[4] / res1[1] * 100.0;
    res[2] = (double)res1[5] / res1[2] * 100.0;

    cout << fixed;
    cout.precision(2);
    cout << "Pontos de Saque: " << res[0] << " %." << endl;
    cout << "Pontos de Bloqueio: " << res[1] << " %." << endl;
    cout << "Pontos de Ataque: "<< res[2] << " %." << endl;

    return 0;
}